presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorPageSourceProvider.java

```diff
@@ -78,6 +92,8 @@ public ConnectorPageSource createPageSource(ConnectorTransactionHandle transacti
     private ConnectorPageSource createPageSource(
             FileSystemContext context,
             UUID shardUuid,
+            Optional<UUID> deltaShardUuid,
```

Comment:
Why don't we just use `Optional.empty` to denote `tableSupportsDeltaDelete` is false. That can save us one flag.